aten/src/ATen/core/Generator.cpp

```diff
@@ -13,4 +21,32 @@ at::Tensor Generator::get_state() const {
   return at::Tensor::wrap_tensor_impl(this->impl_->get_state());
 }
 
+Generator make_generator_for_device(c10::Device device, c10::optional<int64_t> seed) {
```

Comment:
Can you move this function into `torch/` somewhere? I think this is the source of the build issue.

e.g. for now you can put it in `torch/csrc/jit/runtime/`. (I know ideally we probably would like this to be somewhere in ATen, but for now I think it will be a good way to work around the build issue - then we can follow up other build experts on whether there's a good way to put this in ATen.